### ISSUE TITLE
colors with better contrast ratio

### ISSUE THREAD

**Issue by jennschiffer** [bocoup/pombot#76]
i started a PR for a11y at #75 but left out some color changes that should be made. I'm using the tota11y bookmarklet to test the color contrast and it came up with some errors

![screen shot 2016-07-14 at 2 28 10 pm](https://cloud.githubusercontent.com/assets/341305/16850935/43dfd7a4-49cf-11e6-8ccf-b792be07570c.png)

i tested it out with a darker red but it clashes with the icon so let's discuss choosing a more eye-friendly color scheme!

**Comment by Wilto** [comment]
![](http://wil.to/_/yes.gif)

**Comment by iamjessklein** [comment]
I couldn't get tota11y to work but I used Lea Verou's contrast ratio webapp with a new red #B11C04 and it works better:

<img width="1044" alt="screen shot 2016-07-18 at 1 45 32 pm" src="https://cloud.githubusercontent.com/assets/535012/16924589/158bf532-4cee-11e6-8a1a-cd511b78fdd0.png">

**Comment by iamjessklein** [comment]
Also, according the [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG/#visual-audio-contrast) it seems like we could maintain the color but up the font size to  achieve the 3:1 ratio.

> 
> 1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)
> Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
> Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
> Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.

**Comment by jennschiffer** [comment]
![screen shot 2016-07-25 at 10 34 45 am](https://cloud.githubusercontent.com/assets/341305/17105016/71b158f6-5253-11e6-85a0-a3d34c1da2fd.png)

**Comment by iamjessklein** [comment]
👍
